docs/quick-start.md

```diff
@@ -1,84 +1,108 @@
 ---
-sidebar_label: "Quick Start"
-title: ""
+sidebar_label: "Quick start"
+title: "Quick start"
+description: Getting started with Kubewarden, installing the Kubewarden stack and taking care of prerequisites and authentication
+keywords: [Kubewarden, installation, quick start, policyserver, clusteradmissionpolicy, admissionpolicy]
 ---
 
-# Quick Start
 
-The Kubewarden stack is made of the following components:
+The Kubewarden stack comprises:
 
-* An arbitrary number of `ClusterAdmissionPolicy` resources: this is how policies are defined inside Kubernetes
-* An arbitrary number of `PolicyServer` resources: this component represents a Deployment of a Kubewarden `PolicyServer`. The policies defined by the administrators are loaded and evaluated by the Kubewarden `PolicyServer`
-* A Deployment of `kubewarden-controller`: this is the controller that monitors the `ClusterAdmissionPolicy` resources and interacts with the Kubewarden `PolicyServer` components
+* Some `ClusterAdmissionPolicy` resources: this is how policies are defined inside Kubernetes
```

Comment:
I think we can add a similar item for `AdmissionPolicy`

Reply:
Ok, I've done that. Does what I've added correctly describe the tech, the difference in talking about ClusterAdmissionPolicy and AdmissionPolicy? I don't want to be introducing early stage confusion :-). Thanks.

Reply:
Maybe 'defined namespace' is better than 'named namespace' ...